### ISSUE TITLE
[4.0] Random image module

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#17449]
### Steps to reproduce the issue
enable and configure the module but do not  enter a folder

### Expected result
module displayed in the front end

### Actual result
```
Warning: mt_rand(): max(-1) is smaller than min(0) in C:\Apache24\htdocs\cms4\modules\mod_random_image\Helper\RandomImageHelper.php on line 40

Notice: Undefined offset: 0 in C:\Apache24\htdocs\cms4\modules\mod_random_image\Helper\RandomImageHelper.php on line 41

Notice: Trying to get property of non-object in C:\Apache24\htdocs\cms4\modules\mod_random_image\Helper\RandomImageHelper.php on line 42

Notice: Trying to get property of non-object in C:\Apache24\htdocs\cms4\modules\mod_random_image\Helper\RandomImageHelper.php on line 42

Warning: getimagesize(C:\Apache24\htdocs\cms4//): failed to open stream: No such file or directory in C:\Apache24\htdocs\cms4\modules\mod_random_image\Helper\RandomImageHelper.php on line 42

Warning: Division by zero in C:\Apache24\htdocs\cms4\modules\mod_random_image\Helper\RandomImageHelper.php on line 54

Warning: Creating default object from empty value in C:\Apache24\htdocs\cms4\modules\mod_random_image\Helper\RandomImageHelper.php on line 74

Notice: Undefined property: stdClass::$folder in C:\Apache24\htdocs\cms4\modules\mod_random_image\Helper\RandomImageHelper.php on line 76
No Images


```

**Comment by ghost** [comment]
Issue confirmed.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17449">issues.joomla.org/tracker/joomla-cms/17449</a>.</sub>

**Comment by ghost** [comment]
closed as having Pull Request #17472<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17449">issues.joomla.org/tracker/joomla-cms/17449</a>.</sub>
